recorded_future/help.md

```diff
@@ -34,9 +34,7 @@ Example input:
 
 ```
 {
-  "api_key": {
```

Comment:
This one is correct 👍 api_key returns an object called secretKey, we need to add it back

Reply:
@ekim-r7 this comment still needs to be addressed, the example input around the api_key input is correct. It should also be added back to the table on line 31